AFQ/_fixes.py

```diff
@@ -81,3 +81,72 @@ def spherical_harmonics(m, n, theta, phi):
     val *= np.exp(0.5 * (gammaln(n - m + 1) - gammaln(n + m + 1)))
     val = val * np.exp(1j * m * theta)
     return val
+
+
+def orient_by_rois(streamlines, roi1, roi2, affine=None, copy=True):
+    """Orient a set of streamlines according to a pair of ROIs
```

Comment:
a comment explaining what this is overriding and why would be helpful. Looks like other functions in this file do so.